tardis/conftest.py

```diff
@@ -75,22 +80,37 @@ def pytest_unconfigure(config):
             "{1}".format(githash, report_content)
         )
 
-        try:
-            doku_conn = dokuwiki.DokuWiki(
-                url=integration_tests_config["dokuwiki"]["url"],
-                user=integration_tests_config["dokuwiki"]["username"],
-                password=integration_tests_config["dokuwiki"]["password"])
-        except gaierror, dokuwiki.DokuWikiError:
-            print "Dokuwiki connection not established, report upload failed!"
-        else:
-            # Upload report on dokuwiki. Temporary link due to prototyping purposes.
-            doku_conn.pages.append("reports:{0}".format(githash[:7]), report_content)
-            print "Uploaded report on Dokuwiki."
+        # These steps are already performed by `integration_tests_config` and
+        # `doku_conn` fixtures, yet they are to be written here again, because
+        # all the fixtures are teared down and no longer usable, when this
+        # method is being called by pytest.
+        integration_tests_configpath = config.getvalue("integration-tests")
+        if integration_tests_configpath is not None:
+            integration_tests_configpath = os.path.expandvars(
+                os.path.expanduser(integration_tests_configpath)
+            )
+            integration_tests_config = yaml.load(open(
+                    integration_tests_configpath))
+
+            try:
```

Comment:
**DRY**
If you really need to create a connection at two places, import it the second time. Even if it's a fixture, you can call the function like a normal function.